beps/0001-notifications-system/README.md

```diff
@@ -143,6 +148,8 @@ The link is a relative or absolute URL. As an example, it can be used:
 - by an external system to request an action within an asynchronous task
 - by a BE plugin to provide link to other part of the Backstage UI (i.e. to the Catalog)
 
+The additional links are an array of title-URL pairs. They can represent immediate actions on the notification (i.e. yes-no) or lead the user to additional details.
```

Comment:
Keeping an 👀 on this. Feeling it's an alright direction for now, but we'll want to find a good balance between baseline data format and custom payload format. Primary purpose there being the ability to process notifications in the backend in a generic way. In the frontend we could use a custom payload and renderer, and even custom conversions to `WebNotifications`

Reply:
Removing the line, keeping the scope as it is.